### ISSUE TITLE
Can't load/evaluate file using CLJS/Figwheel Main

### ISSUE THREAD

**Issue by alzadude** [BetterThanTomorrow/calva#249]
Steps:
Connect to running nrepl server (for cljs support, select Figwheel Main)
Open a cljc/cljs file (Toggle cljs if cljc)
Press Ctrl+Alt+C, Enter

Expected: 
File is loaded/evaluated

Actual:
``` 
Evaluating file: file.cljc
[36mCompile Exception: [0m[33mCannot open <nil> as a Reader.  [0m[36m[0m
```

**Comment by PEZ** [comment]
Could be a dependency issue. Does the project have the same dependencies as jack-in uses?

**Comment by alzadude** [comment]
> Could be a dependency issue. Does the project have the same dependencies as jack-in uses?

Hi @PEZ here's my `deps.edn` to reproduce the issue:
```
{:paths ["src"]
 :deps
 {org.clojure/clojure {:mvn/version "1.10.1"}
  org.clojure/clojurescript {:mvn/version "1.10.520"}}
 :aliases {:nrepl-server {:extra-deps {cider/cider-nrepl {:mvn/version "0.21.1"}
                                       com.bhauman/figwheel-main {:mvn/version "0.2.1"}
                                       cider/piggieback {:mvn/version "0.4.1"}
                                       ^:depot/ignore org.clojure/tools.namespace {:mvn/version "0.2.11"}}
                          :main-opts ["-m" "nrepl.cmdline"
                                      "--middleware" "[cider.nrepl/cider-middleware,cider.piggieback/wrap-cljs-repl]"]}
           :outdated {:extra-deps {olical/depot {:mvn/version "1.8.4"}}
                      :main-opts ["-m" "depot.outdated.main"]}}}
``` 
My `dev.cljs.edn` (for Figwheel):
```
{:main core
 :target :nodejs}
```
My `core.cljc` (simplest possible):
```
(ns core)
```

**Comment by PEZ** [comment]
Thanks. What's the shadow version?

**Comment by alzadude** [comment]
> Thanks. What's the shadow version?

I'm not using Shadow, only Figwheel Main (on NodeJS)

**Comment by PEZ** [comment]
Is this still an issue? A lot has happened with the jack-in since, so I have some little hope this have been fixed.

**Comment by alzadude** [comment]
Hi @pez,

Ok I've just tried using the same simple project as before, on version 2.0.39:

Steps:
Connect to running nrepl server
Select `Clojure CLI + Figwheel Main`
Open a cljc file (`core.cljc` from above)
Toggle cljs
Press `Ctrl+Alt+C, Enter`

Expected:
File is loaded/evaluated

Actual:
```
Evaluating file: /mnt/d/Projects/calva-issue/src/core.cljc
WARNING: core is a single segment namespace at line 1 /mnt/d/Projects/calva-issue/src/core.cljc

=> nil
```
So connecting to a running nrepl server seems to be working ok now! Both in Windows VSCode where the nrepl server is launched from WSL, and also with the new `Remote - WSL` extension 🙂

Further, with the `Remote - WSL` extension, I can now use jack-in instead of manually connecting. That also seems to be working ok now, using a simplified deps.edn since the Figwheel and Piggieback deps don't need to be specified any more (I selected the `dev` alias when prompted):
```
{:paths ["src"]
 :deps
 {org.clojure/clojure {:mvn/version "1.10.1"}
  org.clojure/clojurescript {:mvn/version "1.10.520"}}
 :aliases {:dev {:extra-deps {^:depot/ignore org.clojure/tools.namespace {:mvn/version "0.2.11"}}}
           :outdated {:extra-deps {olical/depot {:mvn/version "1.8.4"}}
                      :main-opts ["-m" "depot.outdated.main"]}}}
```
So, in conclusion I think this issue can now be closed 🙂

**Comment by PEZ** [comment]
Thanks! You made my day.
